TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/TeleOp.java

```diff
@@ -31,6 +32,13 @@ public class TeleOp extends OpMode {
     private final double jointTargetPos = 0.0;
     boolean scoringATM;
```

Comment:
please sort these out a bit, add lines and organize by type/function, seems kinda random right now which doesn't *Really* matter but it's just a bit of a headache to look at lol